config/sync/core.entity_form_display.node.article.default.yml

```diff
@@ -93,6 +93,11 @@ content:
       match_limit: 10
     region: content
     third_party_settings: {  }
+  url_redirects:
+    weight: 50
+    settings: {  }
+    third_party_settings: {  }
+    region: content
```

Comment:
We have to figure out why this has been added.

Reply:
I'll have a look.